jellyfin_kodi/monitor.py

```diff
@@ -104,21 +104,21 @@ def onNotification(self, sender, method, data):
 
         try:
             if not data.get('ServerId'):
-                raise Exception("ServerId undefined.")
-
-            if method != 'LoadServer' and data['ServerId'] not in self.servers:
+                server = Jellyfin()
```

Comment:
Why remove the exception here?

Reply:
Because that exception throws stuff into the logs which users seem to think is the cause of every issue they have with Kodi but really is just a bad way to handle parameters being empty 

example: https://github.com/jellyfin/jellyfin-kodi/issues/83#issuecomment-534892009